cfme/utils/appliance/__init__.py

```diff
@@ -1855,6 +1856,14 @@ def server_region_string(self):
         return "{} Region: Region {} [{}]".format(
             self.product_name, r, r)
 
+    def get_first_slave_server(self):
```

Comment:
Question: Maybe it will be better to separate this, as get_slave_servers function, and for the first slave use property first_slave_server?